src/views/network/Network.vue

```diff
@@ -153,6 +172,9 @@
             },
             total_requests () {
                 return Object.keys(this.requests).length
+            },
+            search_requests () {
+                return items => items.requests.filter(item => new RegExp(this.search_filter === null ? '' : this.search_filter.trim(), 'i').test(this.requests[item].headers.url.pathname))
```

Comment:
You don't need a regex here, we can have a better performance like this:
(also filtering the requests instead of the sessions)

```javascript
search_requests () {
		const term = this.search_filter && this.search_filter.trim().toLowerCase()
		return item => !term || item.headers.url.pathname.toLowerCase().includes(term)
}
```

Maybe this should be camel cased since it returns a function 🤔